package NYT.NTabletClient.NProto;

////////////////////////////////////////////////////////////////////////////////

message TColumnFilter
{
    repeated int32 indexes = 1;
}

message TReqLookupRows
{
    optional TColumnFilter column_filter = 1;
}

////////////////////////////////////////////////////////////////////////////////
